Makefile

```diff
@@ -368,13 +370,10 @@ $(STAMPS)/android-x86: android-env
 	$(TOUCH_STAMP)
```

Comment:
Could we generate rules for all Android targets?

It seems redundant to have to write explicitly the target in cargo invocations and the target name (e.g. `target/x86_64-linux-android/...).

For example (heavily handwaving):
```Makefile
TARGETS := ...
$(foreach target,$(TARGETS),$(eval \
target/$(target)/...: android-env
   $(CARGO) rustc --target $(target) ...
))
```

Reply:
It may be technically possible, but I'd rather not, for 2 reasons:

1. We'd only be generalizing over 2 of the android targets; `android-x86` is straightforwardly different. IMO YAGNI applies here.
2. I'd rather stay away from `foreach/eval` syntax in this makefile if possible; its intrinsic complexity is high enough already without going there.